src/home/search.vue

```diff
@@ -34,11 +33,10 @@
     button.extra.circle.fill(data-ui="#search2")
       i search
       menu#search2.no-wrap.left.min
-        header.fixed
-          .field.large.prefix.suffix.no-margin
-            i.front(data-ui="#search2") arrow_back
-            input(placeholder="Search text")
-            i.front(data-ui="#search2") close
+        .field.large.prefix.suffix.no-margin.fixed
+          i.front(data-ui="#search2") arrow_back
+          input(placeholder="Search text")
+          i.front(data-ui="#search2") close
```

Comment:
When clicking the close and arrow_back we keep the focus maybe we should remove the focus ?

Reply:
This can be another PR too, like the other changes to do in our search.